perf-test-jenkins/CSVCompare.groovy

```diff
@@ -5,41 +5,69 @@ import org.jboss.windup.utils.model.ExportReportModelToCSV;
 import org.jboss.windup.utils.model.ReportModel;
 
 public class CSVCompare {
-    public static String compare(File directory1, File directory2) {
+
+    public static String compare(File oldOutputs, File currentOutputs) {
         String result = "";
-        File[] csvFiles = directory2.listFiles();
-        for (File file : csvFiles) {
-            if (file.getName().toLowerCase().endsWith(".csv")) {
-                File file1 = new File(directory1, file.getName());
-                if (!file1.exists())
-                    result += "Previous rule points information is missing for: " + file + " (previous directory: " + directory1 + ")\n";
-                else
-                    result += compareCSVFiles(file1, file);
+
+        for (File currentCsv : currentOutputs.listFiles()) {
+            if (currentCsv.getName().toLowerCase().endsWith(".csv")) {
+                File oldCsv = new File(oldOutputs, currentCsv.getName());
+                if (!oldCsv.exists()) {
+                    result += "The CSV output " + currentCsv + " is not present in old outputs " + oldOutputs + "\n";
+                } else {
+                    result += compareCsvFiles(oldCsv, currentCsv);
+                }
             }
         }
+
+        for (File oldCsv : oldOutputs.listFiles()) {
+            if (oldCsv.getName().toLowerCase().endsWith(".csv")) {
+                File currentCsv = new File(currentOutputs, oldCsv.getName());
+                if (!currentCsv.exists()) {
+                    result += "The CSV output " + oldCsv + " is not present in current outputs " + currentOutputs + "\n";
+                }
+            }
+        }
```

Comment:
Maybe this could rather produce an object with two sets?
And then, the formatting could be like:
```
Missing in the new reports:
\t...
\t...
Newly appeared in the new reports:
\t...
\t...
```